docs/questionSharing.md

```diff
@@ -10,13 +8,15 @@ In order for another course to use questions from your course into their assessm
 
 ## Two ways to share: Publicly or through "Sharing sets"
 
-Questions can either be shared publicly, so that anyone can preview the questions (but not the source code) and use them in their course, or you can share questions only to specific other courses using sharing sets.
+Questions can either be shared publicly, so that anyone can preview the questions and use them in their course, or you can share questions only to specific other courses using sharing sets.
```

Comment:
```suggestion
Questions can either be shared publicly, so that anyone can preview the questions and use them in their course, or shared only to specific other courses using sharing sets.
```